src/components/Embed.js

```diff
@@ -4,39 +4,34 @@ export default {
   props: {
     field: {
       type: Object,
-      required: true,
+      required: true
     },
     wrapper: {
       type: String,
-      required:false,
-      default: 'div',
-    },
+      required: false,
+      default: 'div'
+    }
   },
-  render (h, {
-    props, data, children, parent,
-  }) {
-    const { field } = props
+  render(h, { props, data }) {
```

Comment:
According to the link you give and https://vuejs.org/v2/guide/render-function.html#The-Data-Object-In-Depth : 
```js
{
  // Same API as `v-bind:class`, accepting either
  // a string, object, or array of strings and objects.
  class: {
    foo: true,
    bar: false
  },
  // Same API as `v-bind:style`, accepting either
  // a string, object, or array of objects.
  style: {
    color: 'red',
    fontSize: '14px'
  },
  // Normal HTML attributes
  attrs: {
    id: 'foo'
  },
  // Component props
  props: {
    myProp: 'bar'
  },
  // DOM properties
  domProps: {
    innerHTML: 'baz'
  },
  // Event handlers are nested under `on`, though
  // modifiers such as in `v-on:keyup.enter` are not
  // supported. You'll have to manually check the
  // keyCode in the handler instead.
  on: {
    click: this.clickHandler
  },
  // For components only. Allows you to listen to
  // native events, rather than events emitted from
  // the component using `vm.$emit`.
  nativeOn: {
    click: this.nativeClickHandler
  },
  // Custom directives. Note that the `binding`'s
  // `oldValue` cannot be set, as Vue keeps track
  // of it for you.
  directives: [
    {
      name: 'my-custom-directive',
      value: '2',
      expression: '1 + 1',
      arg: 'foo',
      modifiers: {
        bar: true
      }
    }
  ],
  // Scoped slots in the form of
  // { name: props => VNode | Array<VNode> }
  scopedSlots: {
    default: props => createElement('span', props.text)
  },
  // The name of the slot, if this component is the
  // child of another component
  slot: 'name-of-slot',
  // Other special top-level properties
  key: 'myKey',
  ref: 'myRef',
  // If you are applying the same ref name to multiple
  // elements in the render function. This will make `$refs.myRef` become an
  // array
  refInFor: true
}
```

data object seems to be an object which contains objects like props.  But no data object. Because of that we don't understand with Abdel why it's :
```js
render(h, { props, data }) {
```
and not: 
```js
render(h, { props, ...data }) {
```

Reply:
After discussion with Hugo we figured out that it's an object context containing itself 'props' 'data' etc.